app/containers/Avatar.js

```diff
@@ -23,7 +24,17 @@ const Avatar = React.memo(({
 		type, text, size, userId, token, avatar, baseUrl
 	});
 
-	let image = (
+	let image = emoji ? (
+		<Emoji
+			theme={theme}
+			baseUrl={baseUrl}
+			getCustomEmoji={getCustomEmoji}
+			isMessageContainsOnlyEmoji
+			customEmojis
```

Comment:
Do this as `true` default value inside `Markdown/Emoji`.
`{ customEmojis = true }`.